# This expresses the motor currents
float32 steering_servo_current
float32 brakes_servo_current
float32 motor_DC_current
